modules/index-service/src/main/java/org/opencastproject/index/service/resources/list/provider/EventsListProvider.java

```diff
@@ -59,13 +59,18 @@ public class EventsListProvider implements ResourceListProvider {
   public static final String STATUS = PROVIDER_PREFIX + ".STATUS";
   public static final String COMMENTS = PROVIDER_PREFIX + ".COMMENTS";
   public static final String PUBLISHER = PROVIDER_PREFIX + ".PUBLISHER";
+  public static final String ISPUBLISHED = PROVIDER_PREFIX + ".ISPUBLISHED";
 
   public enum Comments {
     NONE, OPEN, RESOLVED;
   }
 
+  public enum IsPublished {
+    YES, NO;
+  }
```

Comment:
This is basically a boolean. So why not use a boolean instead?

Reply:
To be honest I do not have any specific reason for choosing an enum except that I started to build the filter in accordance to the comment filter, therefore I just copy-pasted the existing structure for the comment filter

Reply:
As we discussed, after looking into it again, it turned out that the frontend does require a string, therefore a possible boolean needs to be mapped onto a "yes"/"no" string in the `EventListProvider.java` anyway, which means that working with an enum directly saves the step of mapping the boolean to a string. Additionally the existing filter-architecture is expecting strings so that a boolean might require a change of the already existing architecture in several places.